src/main/java/com/revature/sfadapter/web/AuthController.java

```diff
@@ -2,6 +2,7 @@
 
 import com.revature.sfadapter.services.SFAuthService;
 import com.revature.sfadapter.util.SFWSAccessObject;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.revature.sfadapter.util.SFWSAccessObject'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.revatureops.com/coding_rules#rule_key=squid%3AUselessImportCheck)